pyproject.toml

```diff
@@ -0,0 +1,90 @@
+[build-system]
+requires = [
+  "setuptools>=30.3.0",
+  "wheel",
+  "setuptools_scm[toml]>=3.4"
+]
+build-backend = 'setuptools.build_meta'
+
+[tool.setuptools_scm]
+
+[tool.tox]
+legacy_tox_ini = """
+[tox]
+envlist = py37,coverage-report,typing,lint
+isolated_build = True
+
+[testenv]
+description = Run pytest against {envname}.
+extras =
+    dev
+commands=
+    coverage run -m pytest {posargs}
+
+[testenv:coverage-report]
+description = Compile coverage from each test run.
+skip_install = true
+deps = coverage[toml]>=5.0.2
+depends =
+    py37
+    py38
+commands =
+    coverage combine
+    coverage report
+
+[testenv:typing]
+description = Run mypy.
+extras =
+    dev
+commands =
+    mypy src/safir
+
+[testenv:lint]
+description = Lint codebase by running pre-commit (Black, isort, Flake8).
+skip_install = true
+deps =
+    pre-commit
+commands = pre-commit run --all-files
+
+[testenv:docs]
+description = Build documentation (HTML) with Sphinx.
+extras =
+    dev
+commands =
+    sphinx-build -n -T -b html -d {envtmpdir}/doctrees docs docs/_build/html
+"""
+
+[tool.coverage.run]
+parallel = true
+branch = true
+source = ["safir"]
+
+[tool.coverage.paths]
+source = ["src", ".tox/*/site-packages"]
+
+[tool.coverage.report]
+show_missing = true
+
+[tool.black]
+line-length = 79
+target-version = ['py37']
+exclude = '''
+/(
+    \.eggs
+  | \.git
+  | \.mypy_cache
+  | \.tox
+  | \.venv
+  | _build
+  | build
+  | dist
+)/
+'''
+# Use single-quoted strings so TOML treats the string like a Python r-string
+# Multi-line strings are implicitly treated by black as regular expressions
+
+[tool.isort]
+multi_line_output = 3
+known_first_party = "safir"
+known_third_party = ["aiohttp", "safir", "setuptools", "structlog"]
```

Comment:
safir is listed as both first-party and third-party. Is that intended?

Reply:
aha: https://github.com/asottile/seed-isort-config/issues/15